src/config/app.ts

```diff
@@ -67,10 +68,18 @@ export const MEMO_SHARING_SATS_THRESHOLD = yamlConfig.limits.memoSharingSatsThre
 
 export const ONCHAIN_MIN_CONFIRMATIONS = yamlConfig.onChainWallet.minConfirmations
 // how many block are we looking back for getChainTransactions
-export const ONCHAIN_LOOK_BACK = yamlConfig.onChainWallet.lookBack
-export const ONCHAIN_LOOK_BACK_OUTGOING = yamlConfig.onChainWallet.lookBackOutgoing
-export const ONCHAIN_LOOK_BACK_CHANNEL_UPDATE =
-  yamlConfig.onChainWallet.lookBackChannelUpdate
+const getOnChainScanDepth = (val: number): ScanDepth => {
+  const scanDepth = checkedToScanDepth(val)
```

Comment:
Just a comment - not a blocker. I wish we could do a complete config check at startup to fail early and not when these values get read the first time at runtime.

Reply:
if you like I can update this in the next PR